src/main/java/org/influxdb/dto/Point.java

```diff
@@ -328,6 +328,20 @@ public String lineProtocol() {
     return sb.toString();
   }
 
+  /**
+   * Calculate the lineprotocol entry for a single point, using a specific {@link TimeUnit} for the timestamp.
+   * @param precision the time precision unit for this point
+   * @return the String without newLine
+   */
+  public String lineProtocol(final TimeUnit precision) {
+    final StringBuilder sb = new StringBuilder();
+    sb.append(KEY_ESCAPER.escape(this.measurement));
```

Comment:
We need to synchronize this with #322 
@simon04 can you keep track of this change in you PR, depending on what we merge first.